napari/_tests/test_adding_removing.py

```diff
@@ -120,3 +120,15 @@ def my_custom_callback(event):
     assert len(layer.events.callbacks) == 1
     for em in layer.events.emitters.values():
         assert len(em.callbacks) == 1
+
+
+def test_open_metadata_source_stored(make_test_viewer):
```

Comment:
I think we can just add an additional assert to this test here https://github.com/napari/napari/blob/v0.4.2/napari/components/_tests/test_add_layers.py#L45-L48 rather than making a whole new one. Or if we do make a new one it should at least be in that file and use one of our mocked paths rather than a web url

Test would get an additional line

```python
    assert viewer.layers[0].source == 'mock_path.tif'
```

I think

Reply:
can you address this comment too @haesleinhuepf, then I think it's ready to be approved